letsencrypt/Dockerfile

```diff
@@ -53,7 +52,6 @@ RUN \
         certbot==${CERTBOT_VERSION} \
         certbot-dns-azure==${CERTBOT_DNS_AZURE_VERSION} \
         certbot-dns-cloudflare==${CERTBOT_VERSION} \
-        certbot-dns-cloudns==${CERTBOT_DNS_CLOUDNS_VERSION} \
```

Comment:
This can be included with the newly available version 0.7.0 of certbot-dns-cloudns.